os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -271,65 +221,64 @@ void i2s_set_dma_buffer(i2s_t *obj, char *tx_buf, char *rx_buf,
 
 	if (obj->direction == I2S_DIR_TX) {
 
```

Comment:
I think `obj` and `i2s_idx` should be checked ahead because it is assigned by parameter.